### PR TITLE
Implement widgets

### PR DESCRIPTION
**Pull request by AdamLipscore**
Added 3 Magento widgets, to be used by integration team.
Refactoring of configuration.
Added new fields to request sent to Lipscore.

### CODE REVIEW

File: Helper/Locale.php [hunk 1]
```diff
@@ -2,64 +2,49 @@
 
 namespace Lipscore\RatingsReviews\Helper;
 
-use Lipscore\RatingsReviews\Helper\AbstractHelper;
-
 class Locale extends AbstractHelper
 {
-    protected static $availableLocales = [
+    public const AVAILABLE_LOCALES = [
         'en', 'it', 'no', 'es', 'br', 'ru', 'se', 'cz', 'nl', 'dk', 'ja', 'de', 'fi', 'fr'
```
Comment (maksp-lipscore) [inline Helper/Locale.php:8]: Could you update the list of supported languages? The current list is in invitation[lang] section of the documentation: https://members.lipscore.com/apidoc/v2/purchases/create.en.html

Comment (ivanovaolga) [reply Helper/Locale.php:8]: I think it's out of this PR scope. We should do it but as another update on both extension versions: 1.x and 2.x(the current new widget implementation)

File: Helper/Reminder.php [hunk 2]
```diff
@@ -2,65 +2,83 @@
...
         $name   = $this->purchaseHelper->customerName($order);
-        $lang   = $this->localeHelper->getStoreLocale();
+        $lang   = $this->localeHelper->getLipscoreLocale($order->getStoreId());
         $date   = $this->purchaseHelper->createdAt($order);
+        $customerId = $order->getCustomerId() ? $order->getCustomerId() : '';
 
         return array_merge(
             $couponData,
             [
+                'internal_order_id' => (string) $order->getIncrementId(),
```
[the remark this answers is not in the document]
Comment (ivanovaolga) [reply Helper/Reminder.php:80]: I think it's out of this PR scope. We should do it but as another update on both extension versions: 1.x and 2.x (the current new widget implementation)
Could you please discuss it with Magda and create a card on the board?

**Review by ivanovaolga — DISMISSED** [review]
It's amazing, cool and clear code, so nothing from my side to comment.

**Review by ivanovaolga — APPROVED** [review]
just re-approval

**Pull request merged**